components/reddit/actions/list-comments-in-a-post/list-comments-in-a-post.mjs

```diff
@@ -61,24 +61,6 @@ export default {
         "live",
       ],
     },
-    after: {
-      propDefinition: [
-        reddit,
-        "after",
-      ],
-    },
-    before: {
-      propDefinition: [
-        reddit,
-        "before",
-      ],
-    },
-    limit: {
-      propDefinition: [
-        reddit,
-        "limit",
-      ],
-    },
```

Comment:
It seems like the `limit` parameter of [GET [/r/subreddit]/comments/article](https://www.reddit.com/dev/api/#GET_comments_{article}) correctly limits the number of comments fetched, so we could keep it.

I think most users won't care to set a `limit` if comments are not paginated, so I think removing the prop is fine too.